app/utils/RequestCountry.scala

```diff
@@ -1,13 +1,12 @@
 package utils
 
 import com.gu.i18n.CountryGroup
-import com.netaporter.uri.Uri
-import com.netaporter.uri.dsl._
 import play.api.mvc.Request
 
 object RequestCountry {
   implicit class RequestWithFastlyCountry(r: Request[_]) {
-    def getFastlyCountry = r.headers.get("X-GU-GeoIP-Country-Code").flatMap(CountryGroup.byFastlyCountryCode)
+    def getFastlyCountryGroup = r.headers.get("X-GU-GeoIP-Country-Code").flatMap(CountryGroup.byFastlyCountryCode)
+    def getFastlyCountry = r.headers.get("X-GU-GeoIP-Country-Code").flatMap(CountryGroup.countryByCode)
```

Comment:
why is the above by"fastly"countrycode, but this one is just by code? are they both fastly codes really?